ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-openshift-logging-5.5.yaml

```
@@ -71,7 +71,7 @@ tests:
       SIZE_VARIANT: large
       TAG_VERSION: '@logging5.5'
       TEST_ADDITIONAL: ""
-      TEST_FILTERS: CPaasrunOnly;CPaasrunBoth
+      TEST_FILTERS: ~Disruptive&;~disruptive&
```

```suggestion
      TEST_FILTERS: CPaasrunOnly;CPaasrunBoth;~Disruptive&;~disruptive&
```
I think we still need `CPaasrunOnly;CPaasrunBoth;`

---

It shouldn't make any difference if we specify the tag or not. All of our cases have either of the tags.

---

No, this case https://github.com/openshift/openshift-tests-private/blob/master/test/extended/logging/acceptance.go#L47 doesn't have these tags, and it's designed for OCP testing only.